x-pack/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -158,7 +158,7 @@ export async function executor(
 
   const axiosInstance = axios.create();
 
-  const result: Result<AxiosResponse, AxiosError> = await promiseResult(
+  const result: Result<AxiosResponse, AxiosError<{ message: string }>> = await promiseResult(
```

Comment:
__note:__ needed since `error.data` type by default is `unknown` and it's not possible to extract `message` property without forced typecasting.